src/renderer/browser/components/RequestsProcessor/Invoke/Invoke.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { string, func } from 'prop-types';
 
-export default class Invoke extends React.Component {
+export default class Send extends React.Component {
```

Comment:
Why the need to rename it to `Send`?

Reply:
Err, this was a mistake. 😅

Reply:
@DalderupMaurice Fixed.  I meant to change the name of the class in `Send.js`, as that was called `Invoke`.